grid.v3/client.go

```diff
@@ -66,14 +66,14 @@ func (c *Client) Peers(timeout time.Duration) ([]string, error) {
 // the Serve method to act as a server for the namespace. The context can be
 // used to control cancelation or timeouts.
 func (c *Client) PeersC(ctx context.Context) ([]string, error) {
-	regs, err := c.registry.FindRegistrations(ctx, "grid-"+c.namespace)
+	regs, err := c.registry.FindRegistrations(ctx, c.namespace+"-grid-")
 	if err != nil {
 		return nil, err
 	}
 
 	peers := make([]string, 0)
 	for _, reg := range regs {
-		peers = append(peers, reg.Key)
+		peers = append(peers, reg.Key[len(c.namespace+"-"):])
```

Comment:
I assume we just let it panic if Key doesn't begin with `c.namespace`?  Because that would be a bug in the registry.